flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/local/LocalExecutorITCase.java

```diff
@@ -1618,6 +1620,46 @@ public void testCreateFunctionWithHiveCatalog() throws Exception {
         executor.closeSession(sessionId);
     }
 
+    @Test
+    public void testLoadModule() throws Exception {
+        final Executor executor =
+                createModifiedExecutor(
+                        MODULES_ENVIRONMENT_FILE, clusterClient, createModuleReplaceVars());
+        final SessionContext session = new SessionContext("test-session", new Environment());
+        String sessionId = executor.openSession(session);
+        assertEquals("test-session", sessionId);
+
+        exception.expect(SqlExecutionException.class);
+        exception.expectMessage("Could not execute statement: load module core");
+        executor.executeSql(sessionId, "load module core");
+
+        executor.executeSql(sessionId, "load module hive");
+        assertShowResult(
+                executor.executeSql(sessionId, "show modules"), Arrays.asList("core", "hive"));
```

Comment:
Would be better to execute a query which uses Hive functions after this. This can verify the Hive module is exactly loaded, and this is an integration test.